config/initializers/delayed_job.rb

```diff
@@ -16,12 +16,10 @@ class Delayed::Job
   scope :failed, -> { where("failed_at IS NOT NULL") }
 end
 
-def database_deadlocks_when_using_optimized_strategy?
+database_deadlocks_when_using_optimized_strategy = lambda do
```

Comment:
nope, its a local variable which lives in the scope of the file (does not leak elsewhere).